crates/linear_algebra/Cargo.toml

```diff
@@ -6,9 +6,16 @@ license.workspace = true
 homepage.workspace = true
 
 [dependencies]
-approx = { workspace = true }
+approx = { workspace = true, optional = true }
 nalgebra = { workspace = true }
 num-traits = { workspace = true }
-path_serde = { workspace = true }
-serde = { workspace = true }
+path_serde = { workspace = true, optional = true }
+serde = { workspace = true, optional = true }
 simba = { workspace = true }
+
+[features]
+serde = ["dep:serde"]
+path_serde = ["dep:path_serde"]
```

Comment:
Do we publish this optional dependency as well?
Does not doing so break crates.io?

Reply:
I'm not sure... If that is the case, the only option (besides publishing `path_serde` as well) would be to provide the `path_serde` implementations in the `path_serde` crate...